app/controllers/concerns/metadata.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Executes basic metadata template logic for controllers
```

Comment:
**nitpick (blocking):** Could you update comments and variables and here as this is no longer related to `templates`

Reply:
Updated in 70e64dd